### PR TITLE
bump version to 8.17.1

### PR DESCRIPTION
**Pull request by jasoniangreen**

### CODE REVIEW

**Review by epoberezkin — APPROVED** [review]

**Comment by andylizi** [comment]
Hi, the 8.17.0 release was not uploaded to npm. Would you mind publishing the release there?

Edit: Saw the release. Thanks!

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "ajv",
-  "version": "8.16.0",
```
Comment (KolomboPulse) [inline package.json:3]: 16.0

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
...
+  "version": "8.17.1",
```
Comment (KolomboPulse) [inline package.json:3]: 17.1

**Review by sh471 — COMMENTED** [review]
"version": "8.17.1",

**Pull request merged**